### PR TITLE
820 dm link list in tabs has blue hover and blue text

### LINKED ISSUE #820: DM: Link List in tabs has blue hover and blue text
https://hcaptest.utsa.edu/faculty/business-service-centers/

Image

### PR DESCRIPTION
**Pull request by garza-utsa**
closes #820

### CODE REVIEW

**Review by prd409-utsa — APPROVED** [review]
Looks good!

File: components/02-components/02-tab/tab.scss
Comment (lalov1) [inline components/02-components/02-tab/tab.scss:1]: It looks like removing the `ul` and `ol` list is causing them to be white: 
Screenshot 2024-06-07 at 3 09 21 PM

**Review by lalov1 — CHANGES_REQUESTED** [review]
Please see comments about removal of `ul li, ol li` rule

File: components/02-components/02-tab/tab.scss
Comment (garza-utsa) [reply components/02-components/02-tab/tab.scss:1]: should be fixed with latest commit

**Review by Uri-E — COMMENTED** [review]
There appears to be some white text inside a <pre> tag. Don't know how often that is used, but I guess we need to add it to the list of text styles to turn blue.

**Review by lalov1 — APPROVED** [review]
👍

**Review by Uri-E — APPROVED** [review]

**Pull request merged**